STYLE_API_TEMPLATE.md

```diff
@@ -29,31 +29,34 @@ The following sections describe the basic API documentation structure. Each sect
 Depending on where the documentation appears within a section or subsection, heading levels may be adjusted to fit with other content.
 
 1. Name of API (heading level 2)
-1. (Optional) Path and HTTP methods (heading level 3)
+1. (Optional) Path parameter usage examples (heading level 3)
 1. Path parameters (heading level 3)
 1. Query parameters (heading level 3)
 1. Request fields (heading level 3)
 1. Sample request (heading level 4)
 1. Sample response (heading level 4)
 1. Response fields (heading level 3)
 
-## API name
+## {Name of API} API
```

Comment:
According to the OpenAPI initiative, an HTTP path is the CRUD operations before the endpoint, such as GET, PUT, and Delete. With that in mind, we do need to include the HTTP methods for each API, which is why this header was named "Path and HTTP methods". 

For reference, see the OpenAPI documentation: https://swagger.io/docs/specification/paths-and-operations/

Reply:
Yes. That is correct, however the definition of HTTP methods (GET, PUT, POST, etc.) is not in question.

There are several inaccuracies with the new section title and its contents: 
"Path and HTTP methods" does not describe what we are showing in the section body.
The section contains several API calls with an HTTP method and each of the optional path parameters.

Suggestion to change heading to accurately reflect what is shown in the section: "Path parameter usage."

**Path parameter usage** 

The following examples show various path parameter usage with this API.

```
GET /_nodes
GET /_nodes/<nodeId>
GET /_nodes/<metrics>
GET /_nodes/<nodeId>/<metrics>
GET /_nodes/<nodeId>/info/<metrics>
```

Reply:
But not all endpoints have path parameters. All endpoints do have HTTP operations.

Reply:
correct. then we would not include those in this list of examples, right?

Reply:
just to refocus the issue: 
"Path and HTTP methods" does not describe what we are showing in the section body.
The section contains several API calls with an HTTP method and each of the optional path parameters.

Suggestion to change heading to accurately reflect what is shown in the section: "Path parameter usage."